test/utils/svm/utils.ts

```diff
@@ -364,6 +365,12 @@ export const setCurrentTime = async (signer: KeyPairSigner, solanaClient: RpcCli
   );
 };
 
+export const getCurrentTime = async (solanaClient: RpcClient) => {
+  const statePda = await getStatePda(SvmSpokeClient.SVM_SPOKE_PROGRAM_ADDRESS);
```

Comment:
The SVM Spoke now uses a test feature that retrieves the time from the state account, rather than the actual on-chain time. See [get_current_time](https://github.com/across-protocol/contracts/blob/da166638625b01851cd998920dd3acebc81dfb2d/programs/svm-spoke/src/utils/testable_utils.rs#L37-L47).